added_values.go

```diff
@@ -559,7 +559,7 @@ func (i valueInt64) StrictEquals(other Value) bool {
 }
 
 func (i valueInt64) assertInt() (int, bool) {
-	return 0, false
+	return int(i), true
```

Comment:
[q] I could be missing something, but do we need both `valueInt` and `valueInt64`? Originally `valueInt` was of type `int64`. If we changed that back would we need to support `valueInt64` as well? Throughout goja we also convert `valueInt` to an `int64`

Reply:
yes we do, because of how we do schema / bson conversions we need to keep the types separate. an int64 resolves to a Long bson type whereas the valueInt will resolve to an int bson type. Because of goja's implementation of everything becoming a `valueInt`, we lost the distinction between a Long and an Int in bson. I'm open to suggestions / thoughts on a better way to handle this